modules/dkan/dkan_linkchecker/dkan_linkchecker.module

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/**
+ * @file
+ * Code for the DKAN Link Checker feature.
+ */
+
+include_once 'dkan_linkchecker.features.inc';
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function dkan_linkchecker_preprocess_page(&$vars) {
+  if (arg(2) == 'dkan-linkchecker-report' || arg(4) == 'dkan_linkchecker_reports') {
+    drupal_add_css(drupal_get_path('module', 'dkan_linkchecker') . '/css/linkchecker.css');
+  }
+}
+
+/**
+ * Implements hook_menu().
+ */
+function dkan_linkchecker_menu() {
+  $items = array();
+  // Add links to command center menu for site manager access.
+  $items['admin/dkan/dkan-linkchecker-report'] = array(
+    'menu_name' => 'menu-command-center-menu',
+    'title' => 'Broken Links Report',
+    'page callback' => '_dkan_linkchecker_report',
+    'access arguments' => array('access broken links report'),
+    'weight' => 11,
+    'type' => MENU_NORMAL_ITEM,
+  );
+
+  $items['admin/config/link-checker-settings'] = array(
+    'menu_name' => 'menu-command-center-menu',
+    'title' => 'Link Checker Settings',
+    'page callback' => '_dkan_linkchecker_form',
+    'access arguments' => array('administer linkchecker'),
+    'weight' => 10,
+    'type' => MENU_NORMAL_ITEM,
+  );
+
+  return $items;
+}
+
+/**
+ * Redirect to link checker report.
+ */
+function _dkan_linkchecker_report() {
+  drupal_goto('admin/reports/dkan-linkchecker-report');
+}
+
+/**
+ * Redirect to link checker form.
+ */
+function _dkan_linkchecker_form() {
+  drupal_goto('admin/config/content/linkchecker');
+}
+
+/**
+ * Implements hook_menu_alter().
+ */
+function dkan_linkchecker_menu_alter(&$items) {
+  // Remove normal linkchecker report link.
+  $items['admin/reports/linkchecker']['access callback'] = FALSE;
```

Comment:
I am assuming that this report is the default view you disabled previously. In that case, is this necessary?

Reply:
If you don't disable the original view there will be multiple report pages and confusion in my opinion.

Reply:
What does the code in dkan_linkchecker_enable do?

Reply:
That is removing the link to the disabled view from the admin menu.